sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/samples/SampleSnippets.cs

```diff
@@ -61,7 +55,7 @@ public void CreateCertificate()
             #endregion
         }
 
-        [Test]
+        [Ignore("The certificate was already created in the synchronous method.")]
```

Comment:
Don't we want to run both the sync and async to assure they both work?

Reply:
We can't create the same one simultaneously. We could create a different once and make sure we clean that up, but we already test asynchronously creating certs elsewhere. Making sure this sample works when the sync and async versions of this methods are effectively dups and right next to each other I think is unnecessary.

Reply:
We also ignore one of the deletion variants. This was added to provide a snippet source and make sure we compile. We have pretty good coverage from this file to make sure it works. If the implementation was more than a couple lines, I would agree. I think this is suitable risk to avoid complicating the sample further.